modules/Microsoft.Sql/servers/readme.md

```diff
@@ -41,7 +43,7 @@ This module deploys a SQL server.
 | Parameter Name | Type | Default Value | Description |
 | :-- | :-- | :-- | :-- |
 | `administratorLogin` | string | `''` | The administrator username for the server. Required if no `administrators` object for AAD authentication is provided. |
-| `administratorLoginPassword` | securestring | `''` | The administrator login password. Required if no `administrators` object for AAD authentication is provided. |
+| `administratorLoginPassword` | secureString | `''` | The administrator login password. Required if no `administrators` object for AAD authentication is provided. |
```

Comment:
It appears you're not using the latest Bicep version (0.14.6). With that update the PG (unfortunately, but on purpose) changed the type to `securestring`

Reply:
Thanks! Upgraded.